docs/business-finder-link-based-facets.md

```diff
@@ -0,0 +1,51 @@
+# EU Exit Business Readiness finder link-based facets
+
+This document explains why the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) ("business finder") uses [content-tagger](https://github.com/alphagov/content-tagger) 'links' to filter the finder.
+
+## How the business finder is different to other finders
+
+### From the user perspective
+
+In most finders, search results get more granular as you select values in different facets (the number of results goes down). The business finder search results get more _broad_ as you select values in different facets (number of results goes up).
+
+The business finder can be sorted by topic ("grouped display"). In this context, a 'topic' is the label of the facet (such as 'Personal data'), except for the 'Sector / Business area' facet, where the _facet value_ (such as 'Aerospace') is the topic.
+
+![Screenshot of 'Sort by Topic'](https://user-images.githubusercontent.com/93511/52811122-38eb2500-308c-11e9-843f-4df29b719235.png)
+
+This view only applicable to the business readiness finder, but is not yet a candidate for deletion, as it's been proven to be a useful feature.
+
+### From the content designer perspective
+
+In most finders, facets are published as part of the content item itself. This means when the content item is updated via its publishing application, the facets are updated too and associated with the change note accompanying the change. This triggers an email to subscription lists which are subscribed to that facet, and the change note appears in the email body.
+
+The business finder retrieves content from multiple different publishing apps (via `search-api`), therefore facets are published independently:
+
+- There is a ['facet group' associated with the business finder](https://content-tagger.integration.publishing.service.gov.uk/facet_groups). The facet group has a number of possible facets (e.g. "Sector / Business Area"), each of which have a number of possible facet values (e.g. "Accommodation"), which is a [manually curated list in YAML](https://github.com/alphagov/content-tagger/blob/master/lib/data/find-eu-exit-guidance-business.yml).
```

Comment:
These bullets and the last line "The content will now appear in the search results of the business finder when that facet value is selected." describe the process for tagging content in content-tagger and should be moved there.